handler.js

```diff
@@ -0,0 +1,43 @@
+/* handler.js
+ * Manages tests running in parallel
```

Comment:
Thanks :)

<bikesh>Better not to add the file name because this is not robust to renames</>